http://bugs.gentoo.org/266014

ARM defines SYS_cacheflush but does not have asm/cachectl.h

--- a/system.c
+++ b/system.c
@@ -70,7 +70,7 @@
 #include <linux/capability.h>
 #endif
 
-#ifdef SYS_cacheflush
+#if defined(SYS_cacheflush) && !defined(ARM)
 #include <asm/cachectl.h>
 #endif
 
